### PR TITLE
pangea-node-sdk: updates for AuthZ v1 (GEA-13602)

### PR DESCRIPTION
**Pull request by kenany**
Renamed "namespace" to "type" and changed endpoint paths to `/v1/`.

### CODE REVIEW

File: examples/authz/authz_cycle.mjs [hunk 1]
```diff
@@ -29,45 +29,45 @@ const RELATION_READER = "reader";
       tuples: [
         {
           resource: {
-            namespace: NAMESPACE_FOLDER,
+            user: TYPE_FOLDER,
```
[the remark this answers is not in the document]
Comment (pangea-andrest) [reply examples/authz/authz_cycle.mjs:32]: Same in `subject`?

File: packages/pangea-node-sdk/tests/integration/authz.test.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ it("AuthZ cycle", async () => {
   // Tuple list with resource
   const rListWithResource = await authz.tupleList({
     filter: {
-      resource_namespace: NAMESPACE_FOLDER,
+      resource_namespace: TYPE_FOLDER,
```
Comment (pangea-andrest) [inline packages/pangea-node-sdk/tests/integration/authz.test.ts:89]: Should update this filter?

File: examples/authz/authz_cycle.mjs [hunk 3]
```diff
@@ -79,31 +79,31 @@ const RELATION_READER = "reader";
     console.log("Listing tuples with resource...");
     const rListWithResource = await authz.tupleList({
       filter: {
-        resource_namespace: NAMESPACE_FOLDER,
+        resource_user: TYPE_FOLDER,
```
Comment (pangea-andrest) [inline examples/authz/authz_cycle.mjs:82]:
```suggestion
        resource_type: TYPE_FOLDER,
```

File: examples/authz/authz_cycle.mjs [hunk 4]
```diff
@@ -79,31 +79,31 @@ const RELATION_READER = "reader";
...
+      console.log(`\tType: ${tuple.subject.type}`);
       console.log(`\tSubject ID: ${tuple.subject.id}`);
     });
 
     // Tuple list with subject
     console.log("Listing tuples with subject...");
     const tListWithSubject = await authz.tupleList({
       filter: {
-        subject_namespace: NAMESPACE_USER,
+        subject_user: TYPE_USER,
```
Comment (pangea-andrest) [inline examples/authz/authz_cycle.mjs:98]:
```suggestion
        subject_type: TYPE_USER,
```

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: examples/authz/authz_cycle.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (kenany) [reply examples/authz/authz_cycle.mjs]: Oops. Need type-checking in the examples.

File: packages/pangea-node-sdk/tests/integration/authz.test.ts (hunk 2 above)
Comment (kenany) [reply packages/pangea-node-sdk/tests/integration/authz.test.ts]: Updated.

File: examples/authz/authz_cycle.mjs (hunk 3 above)
Comment (kenany) [reply examples/authz/authz_cycle.mjs]: Find-replace mistake. Fixed.

File: examples/authz/authz_cycle.mjs (hunk 4 above)
Comment (kenany) [reply examples/authz/authz_cycle.mjs]: Fixed.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**